faust/tables/recovery.py

```diff
@@ -706,7 +706,7 @@ async def _build_offsets(
         earliest = await consumer.earliest_offsets(*tps)
         # FIXME To be consistent with the offset -1 logic
         earliest = {
-            tp: offset - 1 if offset is not None else None
+            tp: offset if offset is not None else None
             for tp, offset in earliest.items()
         }
```

Comment:
Of course, you can just remove this lines and not re-define `earliest` at all.

Reply:
I messed around with things here, and either the unittests are flawed or the reason we have the `-1` logic is due to a quirk with `aiokafka`. It'll be a bit before I can get to the bottom of this bug, unfortunately.